packages/dds/matrix/src/test/memory/matrix.spec.ts

```diff
@@ -52,7 +52,7 @@ describe("Matrix memory usage", () => {
 		);
 
 		const numbersOfEntriesForTests = isInPerformanceTestingMode
-			? [100, 1000, 10_000]
+			? [100, 1000, 5000]
```

Comment:
Was it intentional to change the format and remove the _ to denote thousands? I saw some discussion about this on another PR the other day.

Reply:
linter or formatter (not sure which) is opinionated on this matter IIRC.